src/lib/util/myDynlink.ml

```diff
@@ -30,35 +30,41 @@ end
 
 include Dynlink
 
-let load fmt get_verbose p msg =
-  if get_verbose then
-    Format.fprintf fmt "[Dynlink] Loading the %s in %S ...@." msg p;
+let load verbose p msg =
+  Printer.print_vrb ~verbose:verbose
+    "[Dynlink] Loading the %s in %S ..." msg p;
   try
     loadfile p;
-    if get_verbose then Format.fprintf fmt "Success !@.@."
+    Printer.print_vrb ~verbose:verbose
+      "Success !@."
   with
   | Error m1 ->
-    if get_verbose then begin
-      Format.fprintf fmt "[Dynlink] Loading the %s in plugin %S failed!@."
+    if verbose then begin
+      Printer.print_vrb
+        "@.[Dynlink] Loading the %s in plugin %S failed!@."
         msg p;
-      Format.fprintf fmt ">> Failure message: %s@.@." (error_message m1);
+      Printer.print_err
+        ">> Failure message: %s@." (error_message m1);
     end;
     let pp = Format.sprintf "%s/%s" Config.pluginsdir p in
-    if get_verbose then
-      Format.fprintf fmt "[Dynlink] Loading the %s in %S ... with prefix %S@."
-        msg p Config.pluginsdir;
+    Printer.print_vrb ~verbose:verbose
+      "[Dynlink] Loading the %s in %S ... with prefix %S .."
```

Comment:
```suggestion
      "[Dynlink] Loading the %s in %S... with prefix %S"
```

Reply:
I've replace the ending `%S ..` with `%S...` since we're expecting to print success right after